libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -231,65 +263,51 @@ void AP_MotorsHeli_RSC::output(RotorControlState state)
 
             // control output forced to zero
             _control_output = 0.0f;
+
+            // governor is forced to disengage status and reset outputs
+            governor_reset();
+            _autothrottle = false;
+            _governor_fault = false;
             break;
 
         case ROTOR_CONTROL_IDLE:
             // set rotor ramp to decrease speed to zero
             update_rotor_ramp(0.0f, dt);
 
+            // set rotor control speed to engine idle and ensure governor is reset, if used
+            governor_reset();
+            _autothrottle = false;
+            _governor_fault = false;
             if (_in_autorotation) {
                 // if in autorotation and using an external governor, set the output to tell the governor to use bailout ramp
                 _control_output = constrain_float( _rsc_arot_bailout_pct/100.0f , 0.0f, 0.4f);
             } else {
                 // set rotor control speed to idle speed parameter, this happens instantly and ignores ramping
-                _control_output = get_idle_output();
+                if (_cooldown_time > 0) {
+                    _control_output = get_idle_output() * 1.5f;
```

Comment:
I don't know for sure but I trust that Chris has a better handle on this and I think it is worth waiting for a comment from a user rather than adding a parameter.  We have enough of those already.  Someday I hope someone incorporates the ICE library into the tradheli RSC and it handles this type of stuff.  So my vote is to leave it as is.

Reply:
That makes sense.